src/sampling.rkt

```diff
@@ -123,7 +123,9 @@
 
 (define (ival-eval repr fn pt #:precision [precision (*starting-prec*)])
   (let loop ([precision precision])
-    (define exs (parameterize ([bf-precision precision]) (apply fn pt)))
+    (define exs (parameterize ([bf-precision precision]
+                               [*use-mixed-sampling* #t])
```

Comment:
Wait why do we want mixed precision sampling off by default but on in ival-eval?

Reply:
I just was not sure that we need on the analyze stage because we always used only 128bits for that stage

Reply:
Ah. But it's so fast that let's just always have it on.

Reply:
Oh, I see. Eh, I wouldn't worry about this for now, and I'm a _bit_ worried that checking `used-mixed-sampling` is actually pretty slow (parameters in Racket are slow for some reason).

Reply:
Got it, I will remove this flag,
Hmm, our mixing is also kinda iterative, the thing that is going to happen with analyze is probably the following:
each time a new point will come and exponents from the previous run will not make any sense. 
I will leave it like this, probably, it will not be any critical for analyze